benchmarks/child-creation.bench.js

```diff
@@ -15,13 +14,13 @@ const blog = bunyan.createLogger({
   name: 'myapp',
   streams: [{
     level: 'trace',
-    stream: dest
+    stream: fs.createWriteStream('/dev/null')
```

Comment:
I think we have had questions about running tests and such on Windows. Will this work there?

Reply:
oh I'm not sure - but this isn't anything new – I just updated to `pino.destination` because I missed it before